ThingIFSDK/ThingIFSDK/OnboardWithThingIDOptions.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  OnboardWithThingIDOptions.swift
+//  ThingIFSDK
+//
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+/** Optional parameters of [ThingIFAPI.onboard(_:thingPassword:options:completionHandler)](./ThingIFAPI.html#/s:FC10ThingIFSDK10ThingIFAPI7onboardFS0_FTSS13thingPasswordSS7optionsGSqCS_25OnboardWithThingIDOptions_17completionHandlerFTGSqPS_6Target__GSqOS_12ThingIFError__T__T_).
+*/
+public class OnboardWithThingIDOptions {
+    public let layoutPosition: LayoutPosition?
+    public let dataGroupingInterval: DataGroupingInterval?
+
+    /** initializer.
+
+    - Parameter position: GATEWAY | STANDALONE | ENDNODE.
+    - Parameter interval: 1_MINUTE | 15_MINUTES | 30_MINUTES | 1_HOUR | 12_HOURS.
```

Comment:
should be `INTERVAL_1_MINUTE`, `INTERVAL_15_MINUTE`, `INTERVAL_30_MINUTE`, `INTERVAL_1_HOUR` and `INTERVAL_12_HOUR`